src/test/java/core/basesyntax/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static final String HEADER = "fruit,quantity";
+    private static FruitTransactionParser fruitTransactionParser;
+
+    @BeforeClass
+    public static void setUp() {
+        fruitTransactionParser = new FruitTransactionParserImpl();
+    }
+
+    @Test
+    public void parseTransactions_validData_ok() {
+        final List<FruitTransaction> expected = new ArrayList<>();
+        FruitTransaction firstTransaction = new FruitTransaction();
+        firstTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        firstTransaction.setFruit("apple");
+        firstTransaction.setQuantity(10);
```

Comment:
I am not sure if your list expected should be final
You can create constructor in FruitTransaction. Then you can create FruitTransaction objects in more suitable way